configure

```diff
@@ -5397,8 +5405,8 @@ if test "$_build_scalers" = yes ; then
 	fi
 fi
 
-if test "$_mt32emu" = yes ; then
-	echo_n ", MT-32 emulator"
```

Comment:
I believe that these have been removed by mistake

Reply:
Re-added